src/lib/rmt/utils.rb

```diff
@@ -33,29 +33,9 @@ class RMT::Utils
       'password' => ''
     },
     'database' => {
-      'host' => 'localhost',
```

Comment:
That might be needed by the Yast module itself when it invokes `mysql` commands.

Reply:
Although I've tested the package and it seems to work fine :thinking:

Reply:
I'm not sure why it works though, I'd vote for bringing `host` back.

Reply:
Ok I'll take care of that.